### PR TITLE
Adding a guide for Lychee (photo-management)

### PR DESCRIPTION
**Pull request by TheForcer**
I created a quick guide for an installation of the Lychee photo web app. It is pretty straightforward, as it is written in PHP. Any feedback on this would be welcome :)

### CODE REVIEW

File: source/guide_lychee.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ git clone https://github.com/electerious/Lychee.git
+ Cloning into 'Lychee'...
+ remote: Counting objects: 10458, done.
+ remote: Compressing objects: 100% (8/8), done.
+ remote: Total 10458 (delta 3), reused 7 (delta 2), pack-reused 10448
+ Receiving objects: 100% (10458/10458), 5.74 MiB | 8.85 MiB/s, done.
+ Resolving deltas: 100% (7144/7144), done.
+ [isabell@stardust html]$ mv Lychee/* Lychee/.[!.]* . && rmdir Lychee
```
Comment (nichtmax) [inline source/guide_lychee.rst]: Please do this step by step. We're not in a hurry. Also I don't think `.[!.]*` is self explanatory.

File: source/guide_lychee.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Installation
+============
+
+``cd`` to your `document root`_, then clone the latest release of the Lychee installer and move the files to the document root:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ git clone https://github.com/electerious/Lychee.git
```
Comment (nichtmax) [inline source/guide_lychee.rst]: This way you pull the master branch. I think something like 
```
[isabell@stardust html]$ git clone --branch release/42.13.2 https://github.com/electerious/Lychee 
```
would be better. Then add a 

```
----
Tested with Lychee 3.1.6, Uberspace 7.1.1
```
to the [end of your guide](https://github.com/Uberspace/lab/blob/master/source/README.md)

File: source/guide_lychee.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  * your Lychee database name: we suggest you use an additional_ database. For example: isabell_lychee. Enter that, you can leave the prefix field empty.
+
+For the last step you have to enter the username/password you want to use for the Lychee user.
+
+Updates
+=======
+
+You can regularly check the release section of Lychees GitHub_ for any new releases. 
+
+If a new version is available, ``cd`` to your `document root` and do a simple ``git pull``:
```
Comment (nichtmax) [inline source/guide_lychee.rst]: Same here. Don't use the master branch. Use 
```
git pull origin release/66.6.6
```
instead. For releases there's a feed: https://github.com/electerious/Lychee/releases.atom

File: source/guide_lychee.rst (hunk 2 above)
Comment (TheForcer) [reply source/guide_lychee.rst]: Thanks a lot for the feedback @nichtmax . I added the version specific git commands and improved the rest as suggested.

**Pull request merged**